game-state-service

```diff
@@ -112,6 +126,11 @@ class GameStateService(Gio.Application):
           <arg type='s' name='key' direction='in'/>
           <arg type='v' name='value' direction='in'/>
         </method>
+        <method name='Reset'>
```

Comment:
The GSS doesn't have default values or anything like that, so the word "Reset" could be confusing, maybe use "Delete" instead?

Reply:
Sure.